webapp/settings.py

```diff
@@ -19,6 +19,9 @@
 
 ALLOWED_HOSTS = ['*']
 
+USE_X_FORWARDED_HOST = True
```

Comment:
I have no problem with setting this anyway, but out of curiosity, is there anywhere in the application that uses the host setting? I thought maybe [this line](https://github.com/ubuntudesign/www.canonical.com/blob/master/redirects.json#L5) would use it, but it doesn't look like it:

``` bash
$ curl -I http://www.canonical.com/careers/openstack 
HTTP/1.1 302 Found
Location: /careers/cloud-vacancies
```

TBH I'd rather there was nowhere in the app which used it - I like my website applications to be domain agnostic if possible.